tests/steamship_tests/client/operations/test_embedding_index.py

```diff
@@ -1,65 +1,75 @@
 import pytest
 from steamship_tests.utils.fixtures import get_steamship_client
-from steamship_tests.utils.random import random_index
+from steamship_tests.utils.random import random_index, random_name
 
-from steamship import Steamship, SteamshipError
+from steamship import SteamshipError
 from steamship.base import TaskState
-from steamship.data.embeddings import EmbeddedItem, EmbeddingIndex
-from steamship.data.plugin.plugin_instance import PluginInstance
+from steamship.data.embeddings import EmbeddedItem
 
 _TEST_EMBEDDER = "test-embedder"
 
 
-def create_index(
-    client: Steamship = None,
-    plugin_instance: str = None,
-    handle: str = None,
-    fetch_if_exists: bool = None,
-) -> EmbeddingIndex:
-    client = client or get_steamship_client()
-    return EmbeddingIndex.create(
-        client=client,
-        plugin_instance=plugin_instance,
-        handle=handle,
-        fetch_if_exists=fetch_if_exists,
-    )
-
-
 def test_create_index():
     client = get_steamship_client()
-    plugin_instance = PluginInstance.create(client, plugin_handle=_TEST_EMBEDDER)
 
-    index = create_index(
-        client=client,
-        plugin_instance=plugin_instance.handle,
+    index_handle = random_name()
+    index = client.use_plugin(
+        "embedding-index",
+        index_handle,
+        config={"embedder": {"plugin_handle": _TEST_EMBEDDER, "fetch_if_exists": True}},
+        fetch_if_exists=False,
     )
 
     assert index is not None
 
     # Duplicate creation should fail with fetch_if_exists=False
     with pytest.raises(SteamshipError):
-        _ = create_index(
-            client=client,
-            handle=index.handle,
-            plugin_instance=plugin_instance.handle,
+        index = client.use_plugin(
+            "embedding-index",
+            index_handle,
+            config={"embedder": {"plugin_handle": _TEST_EMBEDDER, "fetch_if_exists": True}},
```

Comment:
This config object feels heavy to me. Can we just pass a embedder_instance_handle we retrieve using `use_plugin`?

Reply:
(agree! see other comment-- will be easier for us to make a call when we can experiment w/ both side by side; will show in future pr)